sanic_jwt/blueprint.py

```diff
@@ -46,14 +48,19 @@ def get_token_reponse(request, access_token, output, refresh_token=None):
     if request.method == 'OPTIONS':
         return text('', status=204)
     # try:
-    user = await request.app.auth.authenticate(request, *args, **kwargs)
+    user = request.app.auth.authenticate(request, *args, **kwargs)
+    if inspect.isawaitable(user):
+        user = await user
     # except Exception as e:
     #     raise e
 
     access_token, output = await get_access_token_output(request, user)
 
     if request.app.config.SANIC_JWT_REFRESH_TOKEN_ENABLED:
         refresh_token = await request.app.auth.get_refresh_token(request, user)
+        # refresh_token = request.app.auth.get_refresh_token(request, user)
```

Comment:
Shouldn't this logic mimic the `store_refresh_token` to check 

1. that it is implemented
2. that it `iscoroutinefunction`

Reply:
Yeap, I did miss this one - _and_ realized we still need more tests ...

Reply:
Agreed. My next merge will also have a lot more too.

Reply:
:+1: